projects/angular/src/emphasis/alert/alert.ts

```diff
@@ -50,12 +51,24 @@ export class ClrAlert implements OnInit, OnDestroy {
     private hostElement: ElementRef
   ) {}
 
+  @Input('clrAlertLightweight')
+  get isLight(): boolean {
```

Comment:
```suggestion
  get isLightweight(): boolean {
```

Reply:
as discussed offline this is actually a breaking change as we are changing the public get method name

Reply:
corrected in commit 2224898